packages/legend-studio/src/components/setup/Setup.tsx

```diff
@@ -218,61 +208,78 @@ const CreateProjectModal = observer(() => {
             isLoading={setupStore.createOrImportProjectState.isInProgress}
           />
           <div className="setup-create__form">
-            <div className="panel__content__form">
-              <div className="panel__content__form__section">
-                <div className="panel__content__form__section__header__label setup-create__modal__project__type">
-                  Project Type
-                </div>
-                <div className="panel__content__form__section__header__prompt">
-                  Any project intended for use in production must be a
-                  production project. Prototype projects should only be used as
-                  playgrounds or proofs-of-concept.
+            {allowCreatingNewProject && (
+              <div className="panel__content__form">
+                <div className="panel__content__form__section">
+                  <div className="panel__content__form__section__header__label">
```

Comment:
Why are we using the word "import" here even when creating projects are allowed. 
Doesn't the flow make sense if
canCreateProject
  true: default to create, but allow import as another option
  false: just show import option?